bin/node/runtime/src/lib.rs

```diff
@@ -116,7 +116,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
 	spec_version: 279,
-	impl_version: 0,
+	impl_version: 1,
```

Comment:
Why do we increase `impl_version` only?

Reply:
Following the increase of implementation version in the target commit used for merge